content/articles/dealing-with-python's-missing-values-a-complete-guide/index.md

```diff
@@ -0,0 +1,105 @@
+### The problems outlined below are caused by missing values.
+- Because of this, it becomes more difficult to rule out the null hypothesis during testing.
+- Parameter estimations could be affected if data is lost.
+- The sample's representation may be distorted as a result.
+- Because of this, interpreting the study's results may be more difficult.
+### Types of missing data-types
+There are a number of classifications that can be used depending on the pattern or data that is lacking from the dataset or data.
+- **Missing Completely at Random**
+
+This occurs when the chance of missing data is unrelated to the precise value that will be obtained or the observed replies to a query.
+**Solution:** Deleting rows or columns.
+- **Missing at Random**
+
+Rather than taking into account a single missing value, a cluster of observed responses has a greater impact on the likelihood that an experimenter will receive a missing response.
+
+**Solution:** Imputation of data.
+- **Missing not at Random**
+
+Missing not at Random is the only piece of information that is lacking, other than the previously listed categories. Managing the MNAR data sets is a major annoyance. Modeling the missing data is the only way to get a good approximation of the parameters in this scenario.
+
+**Solution:** Improve dataset find data.
+### Missing values can be classified into the following categories:
+1. Numerical dataset with a continuous variable or attribute
+2. Qualitative or feature variables are also included in the list of quantitative and objective variables.
+### The Different Types of Imputed Information
+A variety of sizes and shapes are offered in the form of imputations. In order to build an accurate model of our application, we must first fill in any data gaps that exist in our dataset. This is one of the techniques for doing so.
+- The term univariate or mean imputation is used when only the target variable is imputed.
+- Linear regression can be used to predict missing values based on other variables, or multivariate imputation can be used to predict missing values based on a single variable.
+- All missing values in the dataset can be imputed once to create an imputed set of values.
+- Several imputations are needed if the same missing values are imputed in different places in the same dataset more than once. This simply requires repeating a single imputation to produce a large number of imputed datasets from a single source dataset.
+### How to fix our dataset's missing data
+There are a variety of approaches to dealing with missing data; we will look at some of them, but first, we will start from the beginning with things like importing libraries.
+```Python
+import pandas as pan
+import numpy as num
+
+dataset = pan.read_csv("IncomeAndGender.csv")
+dataset.head()
+```
+Looking at the dataset's dimensions as a measure of its size:
+```Python
+dataset.shape

```

Comment:
I guess you were trying this out in jupyter notebook/colab.

Make sure to use print statements here